src/pages/Home.jsx

```diff
@@ -17,7 +18,11 @@ const Card = ({ length }) => {
   const [searchParams,] = useSearchParams()
   let filters = searchParams.getAll('filters').length > 0 ? searchParams.getAll('filters') : ["all"]
```

Comment:
what's the use of getAll if you are using the filter's first index to search
`filters = filters[0].split(",")`